sfkit/utils/constants.py

```diff
@@ -5,8 +5,10 @@
 def is_installed(binary: str) -> bool:
     return shutil.which(binary) is not None
 
-
-SFKIT_API_URL = os.environ.get("SFKIT_API_URL", "https://sfkit-website-bhj5a4wkqa-uc.a.run.app/api")
+TERRA_DEPLOYMENT_ENV = os.environ.get("TERRA_DEPLOYMENT_ENV")
+SFKIT_API_URL = os.environ.get("SFKIT_API_URL",
+    f"https://sfkit.dsde-{TERRA_DEPLOYMENT_ENV}.broadinstitute.org/api" if TERRA_DEPLOYMENT_ENV
+    else "https://sfkit-website-bhj5a4wkqa-uc.a.run.app/api")
```

Comment:
Just for this row, suggestion to pass it as an ENV var, but not urgent since it is not a security issue.

Reply:
could you elaborate pls?

Reply:
This line `https://sfkit-website-bhj5a4wkqa-uc.a.run.app/api` , would it be better to pass as a ENV var rather than have it hardcoded.